src/resources/postcss/ian-client.pcss

```
@@ -5,6 +5,7 @@
 	background-size: auto;
 	cursor: pointer;
 	height: var(--tec-spacer-8);
+	margin-left: auto;
 	width: var(--tec-spacer-8);
```

Don't use spacer vars for element dimensions.
```suggestion
	height: 40px;
	margin-left: auto;
	width: 40px;
```